src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.storage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> FRUIT_STORAGE = new HashMap<>();
+
+    public Map<String, Integer> getFruitStorage() {
+        return FRUIT_STORAGE;
+    }
```

Comment:
Returning the `FRUIT_STORAGE` map directly allows external modification, which can lead to unintended side effects. Consider returning an unmodifiable view of the map or providing methods to safely interact with the storage.